installing/installing_disconnected/installing-disconnected-aws.adoc

```diff
@@ -0,0 +1,126 @@
+[id="installing-disconnected-aws"]
+= Installing a cluster on AWS that uses mirrored installation content
+include::modules/common-attributes.adoc[]
+:context: installing-disconnected-aws
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a
+cluster on Amazon Web Services (AWS) using infrastructure that you provide and
+an internal mirror of the installation release content.
+
+[IMPORTANT]
+====
+While you can install a {product-title} cluster by using mirrored installation
+release content, your cluster still requires internet access to use the AWS APIs.
+====
+
+One way to create this infrastructure is to use the provided
+CloudFormation templates. You can modify the templates to customize your
+infrastructure or use the information that they contain to create AWS objects
+according to your company's policies.
+
+.Prerequisites
+
+//* xref:../../installing/installing_disconnected/installing-disconnected-preparations.adoc[Create a mirror registry on your bastion host]
+// and obtain the `imageContentSources` data for your version of {product-title}.
+* Review details about the
+xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
+processes.
+* xref:../../installing/installing_aws/installing-aws-account.adoc#installing-aws-account[Configure an AWS account]
+to host the cluster.
++
+[IMPORTANT]
+====
+If you have an AWS profile stored on your computer, it must not use a temporary
+session token that you generated while using a multi-factor authentication
+device. The cluster continues to use your current AWS credentials to
+create AWS resources for the entire life of the cluster, so you must
+use key-based, long-lived credentials. To generate appropriate keys, see
+link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html[Managing Access Keys for IAM Users]
+in the AWS documentation. You can supply the keys when you run the installation
+program.
+====
+* Download the AWS CLI and install it on your computer. See
+link:https://docs.aws.amazon.com/cli/latest/userguide/install-bundle.html[Install the AWS CLI Using the Bundled Installer (Linux, macOS, or Unix)]
+in the AWS documentation.
+* If you use a firewall, you must
+xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configure it to access Red Hat Insights].
```

Comment:
What's this for?  I wouldn't expect to _need_ access to Insights.  Currently [the firewall docs][1] just talk about Telemetry / updates, and neither of those are critical to install.  [The disabling-Telemetry docs][2] talk about that in more detail, and I'd expect most disconnected folks to disable update downloads and Telemetry and Insights uploads.  Maybe you want to talk about the pros, cons, and methods behind enabling/disabling each of those in the body of the doc, but I don't think we want to make a hard *must* statement here.

[1]: https://github.com/openshift/openshift-docs/blob/6316a44e18eed06c0cf4340d6f265501e8e423ce/installing/install_config/configuring-firewall.adoc
[2]: https://github.com/openshift/openshift-docs/blob/6316a44e18eed06c0cf4340d6f265501e8e423ce/modules/telemetry-consequences-of-disabling-telemetry.adoc

Reply:
@wking, disabling telemetry is one of the optional next steps in the assembly. Do you have a bit more information that I can include to help the disconnected installers appreciate why they should disable telemetry?

Reply:
The current disabling-telemetry docs focus on the pros of telemetry and cons of disabling it.  But if you have no way to connect to telemetry, you don't have to think about it, you should just disable it.  Still good to read about the cons, but no sense in having the telemeter container banging its head against the disconnected network every few minutes for the life of the cluster and alerting about those failures.

Reply:
On the other hand, if you're just using mirrors for auditing/performance, and your cluster can still reach Red Hat's telemetry endpoints, you probably want to keep them enabled.